ui/components/app/hold-to-reveal-button/hold-to-reveal-button.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import HoldToRevealButton from './hold-to-reveal-button';
+
+describe('HoldToRevealButton', () => {
+  const onLongPressed = jest.fn();
```

Comment:
Mind putting this in a `beforeEach`? That way it gets reset each test and doesn't get shared between tests.

Reply:
Fixed in: 270557aff1b25a0ce0135093286afd8739462c8d